dashboard/app/models/user.rb

```diff
@@ -404,7 +404,7 @@ def district_teachers(cohort = nil)
   validates_length_of :username, within: 5..20, allow_blank: true
   validates_format_of :username, with: USERNAME_REGEX, on: :create, allow_blank: true
   validates_uniqueness_of :username, allow_blank: true, case_sensitive: false, on: :create, if: 'errors.blank?'
-  validates_uniqueness_of :username, allow_blank: true, case_sensitive: false, on: :update, if: 'errors.blank? && username_changed?'
+  validates_uniqueness_of :username, case_sensitive: false, on: :update, if: 'errors.blank? && username_changed?'
```

Comment:
Because users can no longer change their usernames to be blank (based on the below change), allow_blank: true is no longer needed